lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java

```diff
@@ -93,13 +97,59 @@ protected DatasetApi(HttpClientWrapper httpClientWrapper, String id, boolean isP
         return httpClientWrapper.postJsonString(path, "", parameters, new HashMap<>(), DatasetPublicationResult.class);
     }
 
+    /**
+     * Edits the current draft's metadata, adding the fields that do not exist yet. If `replace` is set to `false`, all specified
+     * fields must be either currently empty or allow multiple values. Replaces existing data.
+     *
+     * @see [[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]
+     * @param s JSON document containing the edits to perform
+     * @return
```

Comment:
There are a bunch of JavaDoc warnings when you build from the command line. Please resolve those.

Reply:
Strange, I don't get any warnings...

Reply:
You're not getting these?:
```[INFO] --- maven-javadoc-plugin:3.2.0:jar (attach-javadoc) @ dans-dataverse-client-lib ---
[INFO] No previous run data found, generating javadoc.
[INFO]
69 warnings
[WARNING] Javadoc Warnings
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "58" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - @return tag has no arguments.
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "58" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - @return tag has no arguments.
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "58" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - @return tag has no arguments.
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "58" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - @return tag has no arguments.
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "58" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - @return tag has no arguments.
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "58" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "47" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see:illegal character: "45" in "[[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]"
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - @return tag has no arguments.
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:108: warning - Tag @see: reference not found: [[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:121: warning - Tag @see: reference not found: [[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]
[WARNING] /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java:138: warning - Tag @see: reference not found: [[https://guides.dataverse.org/en/latest/api/native-api.html#edit-dataset-metadata]]
[INFO] Building jar: /Users/janm/git/service/dataverse-dans/dans-dataverse-client-lib/lib/target/dans-dataverse-client-lib-0.0.6-SNAPSHOT-javadoc.jar
[INFO]
[INFO] --- maven-install-plugin:2.4:install (defaul```

Reply:
nope

Reply:
You are probably building with Java 11 then. I disabled JavaDoc generation when building with Java 11 for the moment because of an incompatibiliy in the JavaDoc plugin we are using to process Markdown content. Please, build with Java 8 and then resolved above warnings.